### DISCUSSION TITLE
Support for T-Beam V1.2 Board in iGate-Code

### DISCUSSION THREAD

**Discussion by HB9DUT** [lora-aprs/LoRa_APRS_iGate#333] [Q&A] [answered]
Hi, 
is there a possibility to officially support the T-Beam V1.2 Boards in the iGate code?

I have backported the power management library from the tracker code. That works so far only the board finder does not. But this did not work reliably before because my T-Beam Boards were always recognized as "TTGO_LORA_V2".

My code works so far but my C++ knowledge is quite rusty and my code parts are not really nice.

**Comment by peterus** [comment, marked as answer]
I am currently restructuring the code and project to support more boards in the future (a lot of cool stuff/boards) git released recently.
This change will include the removal of the BoardFinder and will include multiple new boards.

Please stay tuned for the new changes. I will sone post and update of the source code.

**Reply by HB9DUT** [reply]
Really cool - Thanks Peter

**Reply by DO4AF** [reply]
@HB9DUT could you kindly share your branch where you updated the power management code?
Maybe an option for me to use this until official update is there for 1.2

**Reply by HB9DUT** [reply]
@DO4AF 
You can simply use this branch:
https://github.com/lora-aprs/LoRa_APRS_iGate/tree/remove_board_finder

Works well for me.

Or you use the attached firmware image ;-) 
[firmware.zip](https://github.com/lora-aprs/LoRa_APRS_iGate/files/13464760/firmware.zip)

**Reply by vk4ypb** [reply]
Hi, This branch fails on my lilygo T-BEAM AXP2102-V1.2 when compiled for tbeam_axp2101 environment. It constantly reboots. 

This occurs when I turn ON use_gps to true. It occurs in the code when running the _ss.begin to setup the GPS port. 

Please see attached console log.
[remove_board_finder_console.txt](https://github.com/lora-aprs/LoRa_APRS_iGate/files/13465909/remove_board_finder_console.txt)

**Comment by Glenn272** [comment]
> Hi, This branch fails on my lilygo T-BEAM AXP2102-V1.2 when compiled for tbeam_axp2101 environment. It constantly reboots.

Did you sort this out in the end?  I have the same T-BEAM AXP2102-V1.2 board and even though the default board is reselected for this board in the latest build, it constantly reboots.
Glenn VK3YY.
